setup.py

```diff
@@ -262,6 +262,7 @@ def run(self):
     entry_points={
         'console_scripts': [
             'borg = borg.archiver:main',
+            'borgmount = borg.archiver:main_mount',
```

Comment:
Looks like mount just passes execution over to `mount.<-t>` if it exists[1]. But if there is already an established convention for this thing, we should follow it — I doubt "nice" fstab entries are important to many.

[1]

    $ cat /usr/bin/mount.something 
    #!/bin/sh

    echo mount.something $@
    $ sudo mount -t something a b
    mount.something a b -o rw